Network/Network.cpp

```diff
@@ -59,6 +59,7 @@ using namespace std;
 #define IARM_BUS_NETSRVMGR_API_stopConnectivityMonitoring "stopConnectivityMonitoring"
 #define IARM_BUS_NETSRVMGR_API_isAvailable "isAvailable"
 #define IARM_BUS_NETSRVMGR_API_getPublicIP "getPublicIP"
+#define IARM_BUS_NETSRVMGR_API_configurePNI "configurePNI"
```

Comment:
Pls update API_VERSION_NUMBER_MINOR macro also to 1 and API_VERSION_NUMBER_PATCH to 0 to match changelog.